packages/storybook/stories/Troubleshooting.stories.mdx

```diff
@@ -80,3 +80,19 @@ You may need an appropriate loader to handle this file type, currently no loader
 This issue will occur when using `@azure/communication-react` with a webpack version < 5.
 The best course of action is to upgrade your webpack version to 5.
 If this is not possible, take a look at [these workarounds](https://github.com/fb55/htmlparser2/issues/1237#issuecomment-1182522861).
+
+## Runtime errors
+
+### Failed to create call agent, already exists for this ACS id
+
+```
+Uncaught (in promise) CallError: CallClient.createCallAgent: Failed to create call agent, call agent for this ACS Id already exists
```

Comment:
this shouldn't happen anymore right? It should be the new console warn message?

Reply:
They might see this error from other functions that are invoked as well like the direct create functions. I think that there is value in providing some information for whats going on here.